skripchenkoveta/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  mid = str.length / 2
+  if str.length.odd?
+    str[mid]
+  else
+    str[mid - 1] + str[mid]
+  end
+end
+
+def run_cli
+  loop do
+    print 'Enter string: '
+    str = gets.chomp
+    break if str == 'exit!'
```

Comment:
When you directly compare something to a _particular_ value, it's better to store this value somewhere in a const and compare with a const. This way, when you use this value multiple times in a project and at some point need to update this value, you won't have to search through your code and update it everywhere, but only update the const value.